models/src/test/scala/coop/rchain/models/SortedParMapSpec.scala

```diff
@@ -61,4 +64,24 @@ class SortedParMapSpec extends FlatSpec with Matchers {
     } should be(true)
   }
 
+  it should "be equal when it is equal" in {
+    val ps = Seq(
+      (
+        Par(),
+        Par(
+          List(),
+          List(),
+          List(),
+          List(),
+          List(),
+          List(GPrivate(ByteString.copyFrom(Array[Byte](0))), GPrivate(ByteString.EMPTY)),
+          List(),
+          List(),
+          AlwaysEqual(BitSet()),
```

Comment:
Please replace this with the output of `Pretty.pretty(ps)`, which will output a less noisy version with named arguments, and will only include non-default values for each field.

Reply:
Did that by using `assertEqual`.

Reply:
That is great! :) But what I had in mind is replacing this code - so that it's easier to read, maintain and modify in the future :)

Reply:
Ouch, totally misread it.